metabrainz.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+"""
+Author : Salvador <salvador.estrella.ortiz@gmail.com>
+Date   : 2022-09-30
+Purpose: Query Metadata from MusicBrainz API
+"""
+
+import argparse
+import requests
+from pprint import pprint
+
+
+# --------------------------------------------------
+def get_args():
+    """Get command-line arguments"""
+
+    parser = argparse.ArgumentParser(
+        description="Retrieve metadata from MusicBrainz API",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "-a",
+        "--artist",
+        help="An artist to search for",
+        metavar="str",
+        type=str,
+        nargs="+",
+        required=True,
+    )
+
+    return parser.parse_args()
```

Comment:
**suggestion:** This part should go in the `cli.py` file

Reply:
Done! thanks for the suggestion.